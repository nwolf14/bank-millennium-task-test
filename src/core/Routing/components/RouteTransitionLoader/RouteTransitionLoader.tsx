import { FC } from 'react';

const RouteTransitionLoader: FC = () => <div>RouteTransitionLoader</div>;

export default RouteTransitionLoader;
