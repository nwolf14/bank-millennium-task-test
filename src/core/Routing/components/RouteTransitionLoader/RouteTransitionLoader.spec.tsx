import { render } from '@testing-library/react';

// components
import RouteTransitionLoader from './RouteTransitionLoader';

describe('RouteTransitionLoader', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<RouteTransitionLoader />);

    expect(asFragment()).toMatchSnapshot();
  });
});
