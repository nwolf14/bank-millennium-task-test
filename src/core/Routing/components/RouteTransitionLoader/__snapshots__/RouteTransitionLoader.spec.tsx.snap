// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteTransitionLoader matches snapshot 1`] = `
<DocumentFragment>
  <div>
    RouteTransitionLoader
  </div>
</DocumentFragment>
`;
